src/App.js

```diff
@@ -1,19 +1,235 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.css';
-import chatMessages from './data/messages.json';
+// import ChatEntry from './components/ChatEntry';
+import ChatLog from './components/ChatLog';
+// import chatMessages from './data/messages.json';
+
+const entries = [
```

Comment:
Instead of hardcoding the data in app.js, we should import the data from messages.json so that this file doesn't become too bloated. If you uncomment line 5, you can access the objects within messages.json with the variable chatMessages.